assets/javascripts/script.js

```diff
@@ -27,6 +27,10 @@ function downloadRepo() {
         });
         
         $.ajax({
+            // FIXME: solve "Can't verify CSRF token authenticity."
+            headers: {
+                'X-CSRF-Token': $('meta[name="csrf-token"]').attr('content')
+            },
```

Comment:
This didn't solve "Can't verify CSRF token authenticity.", should I revert it?

Reply:
Yep, i would remove it if it does not work. Avoids confusion in the future.